ledger/ledger-api-test-tool/src/main/resources/logback.xml

```diff
@@ -6,7 +6,7 @@
             <level>trace</level>
         </filter>
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} %-5level %logger{5}@[%-4.30thread] - %msg%n</pattern>
+            <pattern>%date{"yyyy-MM-dd'T'HH:mm:ss,SSSXXX", UTC} %-5level %logger{5}@[%-4.30thread] - %msg%n</pattern>
```

Comment:
The `,` is the European format, whereas `.` is used in English-speaking places.

As DAML is otherwise in US English, I think we should keep the `.`.

```suggestion
            <pattern>%date{"yyyy-MM-dd'T'HH:mm:ss.SSSXXX", UTC} %-5level %logger{5}@[%-4.30thread] - %msg%n</pattern>
```

Reply:
Good point! Thanks 👍

Reply:
I've changed the format according to @SamirTalwar 's suggestion.
@stefanobaghino-da are you ok with that? :)